test/es6/rlexe.xml

```diff
@@ -1331,4 +1331,12 @@
     <compile-flags>-force:deferparse -args summary -endargs</compile-flags>
   </default>
 </test>
+<test>
+  <default>
+    <files>bug_issue_1496.js</files>
+    <compile-flags>-mic:1 -maxsimplejitruncount:2</compile-flags>
```

Comment:
No, dynapogo works for the test case, but it didn't repro the bug in the first place. And given the flags running dynapogo here has little value.